app/ui/src/app/store/entity/entity.store.ts

```diff
@@ -90,7 +94,9 @@ export abstract class AbstractStore<
       this._current,
       this.changeEvents.pipe(
         filter(event => {
-          return event.id.startsWith(this.currentId);
+          log.debug(`got change event for kind: ${event.kind}, id: ${event.id}, `
+                 + `this store is for kind: ${this.service.kind}, id: ${this.currentId}`);
+          return event.kind === this.service.kind && event.id.startsWith(this.currentId);
```

Comment:
The only concern here I'd have is if an integration-bulletin-board event comes in the old code would reload the associated integration, which would then ensure that any new messages in the integration's bulletin board attribute would be picked up and displayed.  Same goes for integration deployments.  If updating these ultimately results in the associated integration having a change event triggered, then I'm all for this.  And then TBH we may as well suppress change events for deployments/bulletin boards, the UI doesn't maintain any local store of these objects.

Reply:
Easy way to test BTW is to create an integration with an extension, then update the extension.  That should trigger a 'configuration required' type of message on the integration.

Reply:
Yeah I don't think change of `integration-bulletin-board` will trigger a `integration` event.

What alternatives do we have?
 - add a new endpoint for the `integration-bulletin-board` and update integration state based on that, it might be difficult/touches a lot of places?
 - change the condition from `event.kind === this.service.kind` to `event.kind.startsWith(this.service.kind)`, seems a bit smelly, but could work
 - also trigger `integration` event on `integration-bulletin-board` change on the backend, not too difficult, but bit smelly -- I mean why do we have `integration-bulletin-board` event?

I think same arguments that can be made for `integration-bulletin-board` events can be made for `integration-deployment` events.

Reply:
Hmm, found out about `setChangeEventsFilter` that uses `event.kind.startsWith('integration')` in `IntegrationStore`.

So perhaps this filter can be removed as there is already one added from the `IntegrationStore`.

Reply:
Right, the existing code basically did that, it triggered an update when events came in for integrations and related types.

Why the UI has to deal with watching all these types is another question for another time perhaps :-)